### PR TITLE
Suggestions for checkout

### PR DESCRIPTION
**Pull request by apawlik**

### CODE REVIEW

File: 11-checkout.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+## Discussion session
+You will be asked to join one of the discussion sessions (run as a video conference) focusing on the lesson of your choice from the Software and Data Carpentry curriculum. The session will be attended by a small group of instructor trainees like yourself and will be led by an experienced instructor. 
+
+This is an informal discussion for trainees to get more comfortable with the training material, clarify anything that may be unclear, learn (if you are still unsure) how to provide feedback and contribute to the materials.
+
+When working on the lesson, preparing for the discussion session, you can think about the following questions (which may also be used to structure the online discussion session):
+
+1. When working through the lesson did you feel like you would like to change the flow / present the concepts differently? If so, which ones and why?
+2. Was there anything that you found unclear in the material? Did you get stuck anywhere?
+3. What could be the misconceptions that the students could develop when learning this material? (i.e. which bits of the module may be tricky - if poorly explained, they may lead the students to broken mental model)
```
Comment (arokem) [inline 11-checkout.md]: perhaps: "What are some misconceptions..." (avoiding repeating "could")?

Comment (arokem) [inline 11-checkout.md]: Also, the parentheses seem like they could be a point on their own.

File: 11-checkout.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+This is an informal discussion for trainees to get more comfortable with the training material, clarify anything that may be unclear, learn (if you are still unsure) how to provide feedback and contribute to the materials.
+
+When working on the lesson, preparing for the discussion session, you can think about the following questions (which may also be used to structure the online discussion session):
+
+1. When working through the lesson did you feel like you would like to change the flow / present the concepts differently? If so, which ones and why?
+2. Was there anything that you found unclear in the material? Did you get stuck anywhere?
+3. What could be the misconceptions that the students could develop when learning this material? (i.e. which bits of the module may be tricky - if poorly explained, they may lead the students to broken mental model) 
+4. What assumptions about the prior knowledge would have for the students taking this module? If you were to list any prerequsites, what would they be?
```
Comment (arokem) [inline 11-checkout.md]: I would reverse the order of these two sentences and would propose something like: "What prior knowledge do learners need to have before participating in this module" (by "module", do you mean lesson, or this some sub-unit?)

File: 11-checkout.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+4. What assumptions about the prior knowledge would have for the students taking this module? If you were to list any prerequsites, what would they be? 
+
+*Question 1* and *Question 2* should help instructor trainees get settled with the [collaborative lesson development approach](http://gvwilson.github.io/swctasks-website/blog/2014/03/collaborative-lesson-development.html) used by Software and Data Carpentry. Whilst it is natural that teaching from the material created by others may be a challenge at the beginning, it is worth to remember that these materials have been developed collaboratively, by a number of people with different perspectives. Hopefully, this allowed creating materials which are well balanced. The discussion session is an opportunity to clarify anything that you found confusing in the material, discuss the logical flow of the lessons and possible ways of delivering them. It actually is beneficial to all stakeholders involved in Software and Data Carpentry if  ["all of our instructors are (...) using our lessons rather than legacy material of their own."](http://software-carpentry.org/blog/2016/01/eighteen-months-of-progress.html)
```
Comment (arokem) [inline 11-checkout.md]: A couple of suggestions:

"get settled" => "become more comfortable" (assuming that's what you meant).
"it is worth to remember" => "it is worth remembering"
"by a number of people" => "by people"
"allowed creating materials which are ..." => "allows development of materials that are"
"It actually" => "It is also beneficial..."

File: 11-checkout.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+At the discussion session we want to give you more confidence with the material.
+
+*Question 3* at the the discussion session makes a reference back to our Instructor Training and helps you develop more awareness about possible "broken mental" models which students may have or may develop learning the particular module.
```
Comment (arokem) [inline 11-checkout.md]: Either 
     broken "mental models"
or: 
    "broken mental models"
or:
   "broken" mental models

**Comment by apawlik** [comment]
@arokem thanks a lot for reviewing this! I'll try to update ASAP :-)

**Pull request merged**